src/service/fixed-layer.js

```diff
@@ -135,6 +135,7 @@ export class FixedLayer {
       // Apply transform style to all fixed elements
       this.fixedElements_.forEach(e => {
         if (e.fixedNow && e.top) {
+          setStyle(e.element, 'transition', 'none');
```

Comment:
I updated the docs. But the name, I think, is fine: it's natural to disable a custom transition for a fine-control transform.